src/actions/copy-async.ts

```diff
@@ -13,15 +13,17 @@ import type { MemFsEditor } from '../index.js';
 import type { AppendOptions } from './append.js';
 import type { CopyOptions, CopySingleOptions } from './copy.js';
 import { resolveFromPaths, render, getCommonPath, type ResolvedFrom, globify, resolveGlobOptions } from '../util.js';
+import { writeInternal } from './write.js';
+import { copySingle } from './copy.js';
 
 const debug = createDebug('mem-fs-editor:copy-async');
 
 async function applyProcessingFileFunc(
-  this: MemFsEditor,
+  editor: MemFsEditor,
   processFile: CopySingleAsyncOptions['processFile'],
   filename: string,
 ) {
-  const output = await Promise.resolve(processFile!.call(this, filename));
+  const output = await Promise.resolve(processFile!.call(editor, filename));
```

Comment:
processFile `this` needs adjusts.

Reply:
Ah good call, also this wrapper is quite unnecessary, I removed it.